channels/channel.nos/nos2010/chn_nos2010.py

```diff
@@ -1331,6 +1332,81 @@ def update_video_item_live(self, item):
         item.complete = True
         return item
 
+    def create_iptv_streams(self, parameter_parser):
```

Comment:
This one should also be in the chn_class.py with a standard response of an empty list.